predict.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+import time
+import numpy as np
+from termcolor import colored
+import pickle
+import common.py
+import csv
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model_filename", dest="model_filename")
+#parser.add_argument("--test_dataset_size", default=700000, type=int, dest="test_dataset_size")
+args = parser.parse_args()
+print "Test dataset size is %s" % colored(args.test_dataset_size, "red")
+
+test_X, _unused = load_dataset_from_file('test.csv', 700000, False)
+
+model_file = open('args.model_filename.pkl', 'rb')
+model = pickle.load(model_file)
+model_file.close()
+
+predicted_probability_distribution_test = model.predict_proba(test_X)
+
+result = []
+result.append([id,hotel_cluster])
```

Comment:
Space between `id` and `hotel_cluster`. Also rename `id` to `idx` since `id` is taken for a [builtin](https://docs.python.org/2/library/functions.html#id) function.